README.md

```diff
@@ -118,6 +118,13 @@ So for IWYU to function correctly, you need to copy the Clang `include` director
 
 This weirdness is tracked in [issue 100](https://github.com/include-what-you-use/include-what-you-use/issues/100), hopefully we can make this more transparent over time.
 
+Another way to insatall is
```

Comment:
Hmm, actually, this new paragraph seems to imply you can use `make install` to work around all the Clang dependencies described just above. It definitely doesn't.

If anything, we could add a little paragraph at the top that says "To install, use the `install` target", but that's pretty much implied.